meson.build

```diff
@@ -190,6 +190,9 @@ if not capstone_dep.found()
   capstone_dep = capstone_proj.get_variable('capstone_dep')
 endif
 
+# Handle PCRE2
+pcre2_dep = dependency('libpcre2-8', fallback : ['pcre2', 'pcre2_dep'], version: '>=10.42', required: true, static: is_static_build)
```

Comment:
allow also system pcre like we do for openssl

Reply:
You mean the toggle? This there automatically selects the system PCRE2 library if present. And only falls back to the subproject if it cannot find the system one.

Reply:
@ret2libc is this ok? i think this should work in the same way we do for the other projects.

Reply:
I agree with @wargio . Follow the same pattern we use for the other dependencies.